testdata/grammar_outside/mini_complexIL.gap

```diff
@@ -0,0 +1,60 @@
+import rna
+
+input rna
+
+signature sig_foldrna(alphabet,answer) {
+	answer nil(Subsequence); //empty structure
+	choice [answer] h([answer]);
+	answer il(Subsequence, Subsequence, answer, Subsequence, Subsequence); // an internal loop with a closing base-pair
+    answer comp(Subsequence, Subsequence, Subsequence, Subsequence, answer, Subsequence, answer);
+}
+
+algebra alg_count auto count;
+algebra alg_enum auto enum;
+
+algebra alg_mfe implements sig_foldrna(alphabet = char, answer = int) {
+  int nil(Subsequence n) {
+    return 0;
+  }
+  choice [int] h([int] i) {
+    return list(minimum(i));
+  }
+  int il(Subsequence lb, Subsequence lr, int x, Subsequence rr, Subsequence rb) {
+    return x + il_energy(lr, rr);
+  }
+  int comp(Subsequence a, Subsequence lr,  Subsequence lr2, Subsequence lr3,  int x, Subsequence b, int y) {
+    return x+y;
+  }
+}
+
+//algebra alg_dotBracket implements sig_foldrna(alphabet = char, answer = string) {
```

Comment:
multi-line comments would be better

Reply:
or remove this unused algebra definition altogether :-)